pkg/repository/prisma/dbsqlc/workflow_runs.sql

```diff
@@ -738,6 +945,43 @@ SELECT
     sqlc.narg('priority')::int
 RETURNING "id";
 
+-- name: CreateStepRuns :copyfrom
+
+INSERT INTO "StepRun" (
+    "id",
+    "tenantId",
+    "jobRunId",
+    "stepId",
+    "status",
+    "requeueAfter",
+    "queue",
+    "priority"
+) VALUES (
+    $1,
+    $2,
+    $3,
+    $4,
+    $5,
+    $6,
+    $7,
+    $8
+
+);
+-- name: GetInsertedStepRuns :many
+
+SELECT id FROM "StepRun"
+WHERE xmin::text = (txid_current() % (2^32)::bigint)::text
+AND "createdAt" >= @createdAt::timestamp
+ORDER BY id;
+
+-- name: GetStepsForWorkflowVersion :many
+
+SELECT
+    "Step".*  from "Step"
+JOIN "Job" ON "Step"."jobId" = "Job"."id"
```

Comment:
why are we joining job it looks like its not needed for the select or where?

Reply:
ah workflowVersionId is on Job? 

lets disambiguate with "Job". (or shorthand "Job" j, j."workflowVersionId")